document/core/binary/values.rst

```diff
@@ -139,6 +140,8 @@ The auxiliary |utf8| function expressing this encoding is defined as follows:
       \iff & \unicode{10000} \leq c < \unicode{110000} \\
       \wedge & c = 2^{18}(b_1-\hex{F0})+2^{12}(b_2-\hex{80})+2^6(b_3-\hex{80})+(b_4-\hex{80})) \\
       \end{array} \\
+   \end{array} \\
+   \where b_2, b_3, b_4 < \hex{C0} \\
```

Comment:
No, all continuation bytes use the full lower 6 bits for payload while the high two bits must be the tag 10 (which is what the side condition ensures). The WHATWG spec obscures this by giving an imperative algorithm with ad-hoc optimisations instead of a declarative spec.